docs/quickstart.md

```diff
@@ -5,17 +5,15 @@ category: 6580d34ee5e4d00068bf2a1d
 ---
 
 ### Installation
+> 📘 Using Local LLMs?
```

Comment:
Should we move this "Using Local LLMs" block to below the quickstart?

Reply:
Hmm, it is also linked in the Advanced section, I think the quick callout here will help users pause if they are using Local LLMs. Maybe move it once MemGPT is more widely used?